.github/workflows/build-and-test-all.yml

```diff
@@ -50,6 +50,7 @@ jobs:
           restore-keys: auth-ccache-
       - run: inv ci-autoconf
       - run: inv ci-auth-configure
+      - run: inv ci-make-distdir
       - run: inv ci-auth-make-bear  # This runs under pdns/
```

Comment:
Can you run this build from inside the freshly generated distdir? That way we can detect missing files in dist.

Reply:
(which means running configure again, but not autoconf)

Reply:
Done

Reply:
The folder name comes from the BUILDER_VERSION variable that has been fixed in `tasks.py`:

```
def ci_autoconf(c):
    c.run('BUILDER_VERSION=0.0.0-git1 autoreconf -vfi')
```

If the goal is to configure and make from inside that directory, we might need to do it with `FUZZING_TARGETS=no` or do something in addition. I will have a look.

Thank you for the review!